### PR TITLE
Add a datepicker to date fields that haven't one

### PR DESCRIPTION
**Pull request by kehh**
Just cos it makes sense.

### CODE REVIEW

File: js/local/local.js [hunk 1]
```diff
@@ -51,7 +51,7 @@ wam.fixEditorWidth = function() {
 wam.userSort = function() {
 
 	var form = $('.sectionBox>form');
-	var screenId = $('#leftNavSidebar .sf-menu-selected').attr('id');
+	var screenId = $('#leftNavSidebar').find(' .sf-menu-selected').attr('id');
```
Comment (leftclickben) [inline js/local/local.js]: These two lines are equivalent, I'm happy with either.  There is also `$('.sf-menu-selected', $('#leftNavSidebar'))`, which makes more sense when you already have $('#leftNavSidebar') in a variable.

File: js/local/local.js [hunk 2]
```diff
@@ -123,10 +122,18 @@ wam.resize = function() {
 wam.init = function() {
 	wam.fixEditorWidth();
 	wam.userSort();
-	// wam.resize();
 };
 
 
 $(function() {
 	wam.init();
+	$('input.dateBg:not(.hasDatepicker)').datepicker(
```
Comment (leftclickben) [inline js/local/local.js:123]: I'd wrap this in a function, something like wam.initDatepickers(), and call it from wam.init().

File: js/local/local.js (hunk 1 above)
Comment (kehh) [inline js/local/local.js]: phpstorm was grumpy saying that it was inefficient use of selectors. http://stackoverflow.com/questions/12674591/inefficient-jquery-usage-warnings-in-phpstorm-ide

Removed the leading space in the selector anyway

Comment (leftclickben) [inline js/local/local.js]: Hmmm, I thought that warning was to do with using the same selector twice without caching the result in a variable.  Anyway the performance difference in this particular case is going to be completely negligible.

**Pull request merged**